provisioning/service/src/Contract/ContractApiHttp.cs

```diff
@@ -487,143 +165,44 @@ private static void ValidateHttpResponse(ContractApiResponse response)
         {
             if (response.StatusCode >= HttpStatusCode.InternalServerError)
             {
-                throw new ProvisioningServiceClientTransientException(response);
+                throw new ProvisioningServiceClientHttpException(response, true);
```

Comment:
Nit: whenever we can't set param names, for code clarity it's nice to have something like 
```C#
 throw new ProvisioningServiceClientHttpException(response, isTransient: true);
```

Reply:
done